purchases/src/main/kotlin/com/revenuecat/purchases/common/diagnostics/DiagnosticsFileHelper.kt

```diff
@@ -23,32 +21,4 @@ internal class DiagnosticsFileHelper(
     fun isDiagnosticsFileTooBig(): Boolean {
```

Comment:
This is the only method remaining here. I want to finish paywall events before dedicating more time to rewriting the whole thing for diagnostics.